modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -25,7 +25,7 @@ export interface MaximumSpendableOptions {
   maxFeeRate?: number;
   recipientAddress?: string;
   limit?: number;
-  target?: number;
+  target?: number | string;
```

Comment:
Instead of adding `| string` to every place, We can introuce a new `type NumberLike = number | string ` if not already there and relace all places where we make this change
```suggestion
  target?: NumberLike;
```

Reply:
I couldn't find any correct name for this type and I think it would be better to keep type in string | number format to keep it more readable.

Reply:
The suggestion was more like introducing this type if not available, but I think its okay to have it as is also too